packages/SwingSet/src/kernel/kernel.js

```diff
@@ -1064,46 +1081,60 @@ export default function buildKernel(
   }
 
   let processQueueRunning;
-  async function tryProcessDeliveryMessage(message) {
+  async function tryProcessMessage(processMessage, message) {
```

Comment:
Since these arguments come from a function named `getNextMessageAndProcessor`, I think they should be named `processor` and `message`, instead of `processMessage`. (Also, `processMessage` bumps up against our incomplete taxonomy of message types, where I've used all of ("send", "deliver", "delivery", "message") to mean the kind of delivery that has a method name and arguments instead of being a promise resolution, and `processor` doesn't)